.github/workflows/publish.yml

```diff
@@ -16,16 +16,19 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - run: git clone --single-branch --no-tags --depth 1 -b master https://grafanabot:${{ secrets.GH_BOT_ACCESS_TOKEN }}@github.com/grafana/website-sync ./.github/actions/website-sync
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
```

Comment:
You should move the `actions/setup-node` step so it runs before this, `yarn` needs `node` to run